packages/manager/apps/pci-private-network/src/api/data/subnets.ts

```diff
@@ -0,0 +1,29 @@
+import { v6 } from '@ovh-ux/manager-core-api';
+
+type AllocationPools = {
+  start: string;
+  end: string;
+};
+
+export type TSubnet = {
```

Comment:
why do you prefix the type with a "T" ?

Reply:
The main reason is visibility. When we use both interfaces and types for typing, it's a good practice to distinguish between them when we read the code ( `T` prefix for type and `I` prefix for Interface ) . This is also why we added suffixes for components last time, for example.

Reply:
Can you link a source explaining why it's a good practice ?